cmd/aries-agent-rest/startcmd/start_test.go

```diff
@@ -241,9 +250,17 @@ func TestStartCmdWithMissingHostArg(t *testing.T) {
 }
 
 func TestStartAgentWithBlankHost(t *testing.T) {
-	parameters := &agentParameters{server: &mockServer{}, host: "", inboundHostInternal: randomURL(),
-		inboundHostExternal: "", dbPath: "", defaultLabel: "",
-		webhookURLs: []string{}, httpResolvers: []string{}, outboundTransports: []string{}, inboundTransport: ""}
+	parameters := &agentParameters{
+		server:               &mockServer{},
+		host:                 "",
+		inboundHostInternals: []string{randomURL()},
+		inboundHostExternals: []string{},
+		dbPath:               "",
+		defaultLabel:         "",
+		webhookURLs:          []string{},
+		httpResolvers:        []string{},
+		outboundTransports:   []string{},
```

Comment:
Why do we need these empty values? I would remove it.

Reply:
also on L287

Reply:
[agreed] - that was an existing code and I didn't focus on it. [Removed](https://github.com/hyperledger/aries-framework-go/compare/1f9c8c18c2255f071b6aa295fbde2d85e23aa7d2..ca59224205e2d944dea6c38a431bfa561f3e7e2c) now.